specviz/widgets/workspace.py

```diff
@@ -357,25 +359,81 @@ def compose_filter_string(reader):
 
         return filters, loader_name_map
 
-    def _on_load_data(self):
-        """
-        When the user loads a data file, this method is triggered. It provides
-        a file open dialog and from the dialog attempts to create a new
-        :class:`~specutils.SpectrumList` object and thereafter adds the
-        contents to the data model.
-        """
+    def _choose_file_path(self):
```

Comment:
Docstrings.

Reply:
So we need docstrings for all purely internal private methods as well?

Reply:
Ah, no, you're right. We don't need it there unless for explanation.